### DISCUSSION TITLE
Doris shows error when connecting to MaxComputer

### DISCUSSION THREAD

**Discussion by chufa** [apache/doris#30634] [A - General / Q&A]
添加mc catalog数据源
show catalogs;  可以正常出来mc
show databases from mc; 可以正常出来database列表
show tables from mc.xx;  异常

报错：[HY000][1105] RuntimeException, msg: org.apache.doris.common.UserException: errCode = 2, detailMessage = java.net.UnknownHostException: service.http

**Comment by LemonLiTree** [comment]
which doris version？
